platformio/commands/access.py

```diff
@@ -38,36 +38,36 @@ def cli():
     pass
 
 
-def validate_urn(value):
+def validate_prn(value):
     value = str(value).strip()
-    if not re.match(r"^reg:pkg:(\d+)$", value, flags=re.I):
-        raise click.BadParameter("Invalid URN format.")
+    if not re.match(r"^reg:pkg:(\d+):(\w+)$", value, flags=re.I):
+        raise click.BadParameter("Invalid PRN format.")
     return value
 
 
 @cli.command("public", short_help="Make resource public")
 @click.argument(
-    "urn", callback=lambda _, __, value: validate_urn(value),
+    "prn", callback=lambda _, __, value: validate_prn(value),
 )
-@click.option("--urn-type", type=click.Choice(["urn:reg:pkg"]), default="urn:reg:pkg")
-def access_public(urn, urn_type):
+@click.option("--prn-type", type=click.Choice(["prn:reg:pkg"]), default="prn:reg:pkg")
```

Comment:
`@click.option("--urn-type", type=click.Choice(["`"]), default="prn:reg:pkg")`

In this case, `urn-type` by default value is `prn:reg:pkg`. The same like URN = URL, and `prn:reg:pkg` = prn://reg.pkg.com (example)